case-projector/src/main/java/eu/itesla_project/case_projector/CaseProjectorTool.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2017, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package eu.itesla_project.case_projector;
+
+import com.google.auto.service.AutoService;
+import eu.itesla_project.commons.config.ComponentDefaultConfig;
+import eu.itesla_project.commons.tools.Command;
+import eu.itesla_project.commons.tools.Tool;
+import eu.itesla_project.computation.local.LocalComputationManager;
+import eu.itesla_project.iidm.import_.Importers;
+import eu.itesla_project.iidm.network.Network;
+import eu.itesla_project.iidm.network.StateManager;
+import eu.itesla_project.loadflow.api.LoadFlowFactory;
+import eu.itesla_project.simulation.SimulatorFactory;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+/**
+ * @author Geoffroy Jamgotchian <geoffroy.jamgotchian@rte-france.com>
+ */
+@AutoService(Tool.class)
+public class CaseProjectorTool implements Tool {
+
+    @Override
+    public Command getCommand() {
+        return new Command() {
+            @Override
+            public String getName() {
+                return "case-projector";
+            }
+
+            @Override
+            public String getTheme() {
+                return "Computation";
+            }
+
+            @Override
+            public String getDescription() {
+                return "run case projector";
+            }
+
+            @Override
+            public Options getOptions() {
+                Options options = new Options();
+                options.addOption(Option.builder().longOpt("case-file")
+                        .desc("the case path")
+                        .hasArg()
+                        .argName("FILE")
+                        .required()
+                        .build());
+                options.addOption(Option.builder().longOpt("generators-domains-file")
+                        .desc("the generators domains file path")
+                        .hasArg()
+                        .argName("FILE")
+                        .required()
+                        .build());
+                return options;
+            }
+
+            @Override
+            public String getUsageFooter() {
+                return null;
+            }
+        };
+    }
+
+    @Override
+    public void run(CommandLine line) throws Exception {
```

Comment:
This method is deprecated: void run(CommandLine line, ToolRunningContext context)